src/classes/STG_PanelRD_CTRL.cls

```diff
@@ -94,39 +84,79 @@ public with sharing class STG_PanelRD_CTRL extends STG_Panel {
         }
     	return null;
     }
-    
+
     /*********************************************************************************************************
     * @description Deletes future Opportunities that are beyond the number of forecast months
-    * @return void 
+    * @return void
     */
     private void trimOpportunities(){
         date limitDate = system.today().addMonths((integer)STG_Panel.stgService.stgRD.npe03__Opportunity_Forecast_Months__c);
         string openlabel = system.label.npe03.RecurringDonationOpenStatus;
-        delete [select id from Opportunity 
-               where CloseDate > :limitDate 
-               and isClosed != true 
+        delete [select id from Opportunity
+               where CloseDate > :limitDate
+               and isClosed != true
                and npe03__Recurring_Donation__r.npe03__Open_Ended_Status__c = :openLabel];
-               
+
         map<Id, npe03__Recurring_Donation__c> mapIdRD = new map<Id, npe03__Recurring_Donation__c>([select Id from npe03__Recurring_Donation__c where npe03__Open_Ended_Status__c = :openLabel]);
-        RD_RecurringDonations.updateRecurringDonationOnOppChangeFuture(mapIdRD.keySet());               
+        RD_RecurringDonations.updateRecurringDonationOnOppChangeFuture(mapIdRD.keySet());
     }
-        
+
     /*********************************************************************************************************
-    * @description The SelectOption list of RD Close Options 
+    * @description set original forecast months for legacy RDs, if added months, start the batch job
+    * @return void
+    */
+    private void setOriginalForecastMonthsForLegacyRD() {
```

Comment:
- The method should be above `trimOpportunities()` (the "calling" method is written above the "called" method).
- Let's give more info in the description: `* @description Sets Opportunity forecast months.  If the new settings is higher, starts the batch job to update RDs. The settings is applicable to legacy Recurring Donations only.`